symex-evil.el

```diff
@@ -49,6 +49,50 @@
   :enable (normal)
   :exit-hook (symex-exit-mode))
 
+(defun symex-evil-repeat-pre-hook-advice (&rest _)
```

Comment:
nit: since this advice function does a single thing, my preference would be to name it after what it does. E.g. `symex-evil-repeat-start-recording-advice`